src/transformers/utils/fx.py

```diff
@@ -760,14 +760,15 @@ def __init__(self, autowrap_modules=(math,), autowrap_functions=()):
             )
 
     def _generate_dummy_input(
-        self, model: PreTrainedModel, input_name: str, shape: List[int]
+        self, model: PreTrainedModel, input_name: str, shape: List[int], input_names: List[str]
```

Comment:
Do we need both `input_name` and `input_names`?

Reply:
Yes. The attention_mask shape will depend on whether `past_key_values` is an input.

Reply:
Let's maybe find better variable names 😅